### PR TITLE
Dingaaling/bigcodecasestudy

### PR DESCRIPTION
**Pull request by dingaaling**
### Summary

Adds BigCode as a case study for a real world example of ML data governance.

### List of changes proposed in this PR (pull-request)

* Add big code case study file to live within the data governance chapter

### What should a reviewer concentrate their feedback on?

- [ ] Technical details look right
- [ ] Grammar/formatting
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by da5nsy** [comment]
This is failing CI because it hasn't been added to the toc.

https://the-turing-way.netlify.app/community-handbook/contributing/contributing-workflow.html#add-your-new-files-to-the-book-s-table-of-contents

**Comment by aleesteele** [comment]
Hi @dingaaling! As @da5nsy mentioned, the file needs to be in the TOC to be rendered. Each file requires both a title and file label. (I actually just had this happen on #3262) 

In other words this:
```
- title: Data Governance for the Machine Learning Pipeline
                file: project-design/data-governance/data-gov-ml
                file: project-design/data-governance/bigcode_casestudy
```
May turn into this:
```
- title: Data Governance for the Machine Learning Pipeline
                file: project-design/data-governance/data-gov-ml
- title: Big Code Case Study
                file: project-design/data-governance/bigcode_casestudy
```
Hope that helps!

**Comment by aleesteele** [comment]
Just checking through the errors - you may need to rebase this branch, as it's showing some errors I'm not sure about during the build 🤔

File: book/website/project-design/data-governance/bigcode_casestudy.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+(pd-dg-bigcode)=
+
+# Case Study: BigCode Data Governance
+_This case study was adapted from the [BigCode Governance card](https://huggingface.co/datasets/bigcode/governance-card) in May 2023._
```
Comment (kallewesterling) [inline book/website/project-design/data-governance/bigcode_casestudy.md:4]: Could this be done as an `{admonition}` or so, to make it stand out a bit from the rest of the text?

**Review by kallewesterling — APPROVED** [review]
This looks great to me. I don't know if case studies on TTW are written in a "us/we/the team" format, but I suggest, if we have incorporated language from elsewhere (which this looks like it did?) that we add an admonition, which would be nice. Let me know if you need help/clarification!

File: book/website/project-design/data-governance/bigcode_casestudy.md (hunk 1 above)
Comment (dingaaling) [reply book/website/project-design/data-governance/bigcode_casestudy.md]: Thank you for reading through and the thoughtful comments, @kallewesterling!

**Pull request merged**